linalg/sundials.hpp

```diff
@@ -948,6 +949,27 @@ class KINSolver : public NewtonSolver, public SundialsSolver
        maximum. */
    void SetMAA(int maa);
 
+   /// Set the damping parameter for KIN_FP or KIN_PICARD.
+   /** The default is 1.0
+       The value of beta must be 0 < beta < 1.0. */
```

Comment:
```suggestion
       The value of beta must be 0 <= beta <= 1.0. */
```
We should fix the bounds in the methods below as well.

Reply:
?

1.0 means no damping and 0.0 means no acceleration. Both values can be set but should be avoided and rather not set at all.

Reply:
Actually, we could simply just updated the first line to `The default is 1.0 (no damping)`.